src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -188,6 +198,19 @@ the main body of the message as well as a quote.
 				<span>{{ t('spreed', 'Unread messages') }}</span>
 			</div>
 		</div>
+
+		<Modal
+			v-if="modal"
+			container="#content-vue"
+			@close="modal=false">
+			<div>
+				<RoomSelector
+					:show-postable-only="true"
+					:dialog-title="setDialogTitle"
+					@select="setSelectedRoom(room)"
```

Comment:
This doesn't work because you're calling the function here, you should remove the `(room)` part and just accept the room as a parameter where you define the event handler

Reply:
@marcoambrosini  Thanks for your answer. But the button has no effect:
![image](https://user-images.githubusercontent.com/24530680/124896960-27e53a80-dfde-11eb-9525-3f1149543564.png)

Tried @select="setSelectedRoom()" and @select="setSelectedRoom"

I still get no log here:
https://github.com/nextcloud/spreed/blob/1e8abdc890b0960297bc0cd4c1a79761d3ecc15f/src/components/MessagesList/MessagesGroup/Message/Message.vue#L801-L805